.github/ISSUE_TEMPLATE/documentation_improvement.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Documentation improvement
+about: Request a documentation update
```

Comment:
"update" -> "improvement" to be consistent with the line above.

Reply:
Done.